CHANGELOG.rst

```diff
@@ -1,5 +1,29 @@
 .. _changelog:
 
+0.46.1
```

Comment:
This can only be `0.45.3` or `0.46.0`.

Reply:
I think it should be `0.46.0` as it's still under preparation. I mean, I added my first commit to it only end of last week and no release/tag has been done for it, yet.